src/DurableTask.AzureStorage/OrchestrationMemoryManager.cs

```diff
@@ -78,15 +79,16 @@ public async Task StopAsync()
                 await this.currentMemoryUpdaterTask;
             }
 
+            this.currentMemoryCancellationTokenSource.Dispose();
```

Comment:
I recommend we guard against null-refs here.
```suggestion
            this.currentMemoryCancellationTokenSource?.Dispose();
```